.github/workflows/rust.yml

```diff
@@ -96,10 +96,11 @@ jobs:
           target: x86_64-unknown-linux-gnu
           cargo-hack-feature-options: --feature-powerset
           additional-deb-packages: libudev-dev
-        # - os: ubuntu-latest-16-cores
-        #   target: aarch64-unknown-linux-gnu
-        #   cargo-hack-feature-options: --feature-powerset
-        #   additional-deb-packages: libudev-dev libssl-dev
+        # TODO: add back ARM support
```

Comment:
should we try this and see if it works?

Reply:
I wanted to test it, but currently it's blocked by us having git dependency in the soroban client 
> it has a git rev dep on the rpc client, which is incompatible with releasing.

(From DMs with Leigh) 
I will check back on it once we release the new client version

Reply:
Hmm, this shouldn't be blocked by a git dep. Git deps should work just fine for the rust workflow. It's the publish dry run workflow that we should expect to error when a git dep is present since we can't publish with git deps.

Reply:
🤦🏻 Oh right, this is the publish dry run. Either way, this should be fine to add back in. The publish dry run is failing for a different reason, but it should pass when the git dep is removed. Fair to keep it commented for now until we can confirm.

Reply:
BTW I think this should be moved out to a different workflow 🤔